resources/functionMap.php

```diff
@@ -121,7 +121,7 @@
 'AMQPEnvelope::getHeader' => ['mixed', 'headerName'=>'string'],
 'AMQPEnvelope::getHeaders' => ['array<string,mixed>'],
 'AMQPEnvelope::getMessageId' => ['string|null'],
-'AMQPEnvelope::getPriority' => ['int'],
+'AMQPEnvelope::getPriority' => ['int<0,max>'],
```

Comment:
AMQP spec defines that a priority MAY be 0-10, where 0 is "default", RabbitMQ supports up to 255, this value is capped to the maximum supported hence leaving it leniant.